package_mod.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# /* this initializes the values for the different variables referenced in compatible_games */
+source "../ModPackaging/utilities/ie_games.sh"
+
+# /* MODIFY: set the values of the 3 variables below to reflect the current mod version */
+export mod_name="Real Mod"
+export mod_version="v2"
+export mod_folder="real_mod"
+
+# /* MODIFY: list here which IE games the mod is compatible with, from this list of possibilities: */
+# /* ${bg1}, ${bg2}, ${bgt}, ${tutu}, ${bgee}, ${bgiiee}, ${iwd1}, ${iwdee}, ${iwd2}, ${iwd2ee}, ${iwd-in-bg2}, ${pst}, ${pstee}, ${eet} */
+export compatible_games="${bg1}, ${bg2}, ${bgt}, ${tutu}, ${bgee}, ${bgiiee} or ${eet}"
```

Comment:
this looks unused and would only work in the readme.

Reply:
It is, currently.  The .bat file uses it for including some text in the self-extracting rar.  I'll check if something like that can be done with zip files.  And if not, then I guess we can eliminate this whole blurp for Linux.

Reply:
Sorry it took me so long to get back on this one. The unused variable has been removed now.